index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+    
+  </head>
+
+  <body>
```

Comment:
In this part of your code, it would be better, in order to have a well-organized code, if you don't leave any blank space between the lines.